sonic_data_client/non_db_client.go

```diff
@@ -246,6 +255,20 @@ func getProcStat() ([]byte, error) {
 	return b, nil
 }
 
+func getSysUptime() ([]byte, error) {
+    timeInfo, _ := linuxproc.ReadUptime("/proc/uptime")
+    uptime := sysUptime{}
+    uptime.Total = uint64(timeInfo.Total)
```

Comment:
I think it can simply marshal and return whatever `linuxproc.ReadUptime` gives.
Do we need to convert this to another struct?

Reply:
The`linuxproc.ReadUptime("/proc/uptime")` will return a struct which contains two `float` values: one value is system uptime, the other is the idle time which all CPUs are in the idle state. I sent an email to Pradyna and Hui. Hui confirmed that currently we are only interested in `system uptime` and do not care about `idle time`. Also we need convert the `float` to `int`.

Reply:
@yozhao101 can you please show sample output of linuxproc.ReadUptime("/proc/uptime")? I tend to agree with Murat here because we can always discard from collector instead here to avoid any regression.

Reply:
@pra-moh The following is the sample output of linuxproc.ReadUptime("/proc/uptime"). Please review it.

admin@str-a7050-acs-3:~$ ./gnmi_cli -client_types=gnmi -a localhost:8080 -t OTHERS -logtostderr -insecure -qt p -pi 10s -q platform/sysuptime
{
  "OTHERS": {
    "platform": {
      "sysuptime": "{\"total\":2986.02,\"idle\":10512.98}"
    }
  }
}
{
  "OTHERS": {
    "platform": {
      "sysuptime": "{\"total\":2996.02,\"idle\":10547.75}"
    }
  }
}
{
  "OTHERS": {
    "platform": {
      "sysuptime": "{\"total\":3006.02,\"idle\":10583.74}"
    }
  }
}

Reply:
Thanks Yong. Let's return total and idle values as they are. As Murat suggested just marshal and return don't do any conversion. We can handle conversion/discarding idle at collector side

Reply:
Sure, I will update.